backend/tracim_backend/error_code.py

```diff
@@ -0,0 +1,56 @@
+# Error code format : [TYPE:2c]-[NUMBER:2c]
+# TYPE:
+# NF=Not found error
+# RI=Right error
+# VA=validation post-marshallow
+# CO=Conflict error
+# AT=Auth error
+# MV = marshmallow schema validation error
+
+ERROR_CODE_GENERIC_SCHEMA_VALIDATION_ERROR = 'SV-01'
+# Tracim Not found Error
+ERROR_CODE_USER_NOT_FOUND = 'NF-01'
+ERROR_CODE_WORKSPACE_NOT_FOUND = 'NF-02'
+ERROR_CODE_CONTENT_NOT_FOUND = 'NF-03'
+# Preview Based
+ERROR_CODE_UNAIVALABLE_PREVIEW_TYPE = 'NF-11'
+ERROR_CODE_PAGE_OF_PREVIEW_NOT_FOUND = 'NF-12'
+ERROR_CODE_UNAIVALABLE_PREVIEW = 'NF-13'
+
+# Right Error
+ERROR_CODE_INSUFFICIENT_USER_PROFILE = 'RI-01'
+ERROR_CODE_INSUFFICIENT_USER_ROLE_IN_WORKSPACE = 'RI-02'
+
+# Auth Error
+ERROR_CODE_NOT_AUTHENTICATED = 'AT-01'
+ERROR_CODE_USER_NOT_ACTIVE = 'AT-02'
```

Comment:
This error case should be shown as «invalid user/pass», not «user deactivated»

Reply:
Done.